<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.stehno.vanilla.test.FixtureBuilderSpec" tests="10" skipped="0" failures="0" errors="0" timestamp="2015-10-23T17:24:54" hostname="RSNPLALT285" time="0.058">
  <properties/>
  <testcase name="fields" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.007"/>
  <testcase name="maps" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.002"/>
  <testcase name="maps: with extra attrs" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.001"/>
  <testcase name="maps: non-existent key" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.007"/>
  <testcase name="maps: empty call on default fix" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.005"/>
  <testcase name="objects" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.009"/>
  <testcase name="objects: with extra attrs" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.002"/>
  <testcase name="verifications" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.006"/>
  <testcase name="verifications: with extra attrs" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.001"/>
  <testcase name="fix with PropertyRandomizer" classname="com.stehno.vanilla.test.FixtureBuilderSpec" time="0.017"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
